### ISSUE TITLE
Make Heavy Duty Alloy T7 Recipes for Galaxy Space

### ISSUE THREAD

**Issue by Dream-Master** [GTNewHorizons/NewHorizons#112]

**Comment by Drobac** [comment]
for Tier 4 we need a recipe for compressed ice ingot and for tier 5 a replacement for lead that can be found only on tier 4 planets, need to check requirements for tier 6

**Comment by Dream-Master** [comment]
But for this we need  Gt ore support on these Planets or I have to make own ore blocks.

**Comment by Drobac** [comment]
will check for a replacement that already exists on one of the planets

**Comment by Drobac** [comment]
Tier V still uses quantium that can be found on the Tier IV planets so that is ok for tier V, if neutronium will be on tier V planets we can use that for the Tier VI  Heavy duty plates

**Comment by Dream-Master** [comment]
I am on it but need time to do it. Lower priority sorry @Drobac

**Comment by Drobac** [comment]
no point in having it untill you can set up the ore gen on the planets so i need to wait on blood esp anyway. Also what kind of eu/t did you mean to put these ?

**Comment by Dream-Master** [comment]
Not sure have to do Oregen first and than the recipes. I add a tone of new stuff so have to see what eu it will cost. It will be fu***** expencive ;-)

**Comment by Dream-Master** [comment]
Tier 4 Is done.

Missing Tier 5-6

**Comment by Dream-Master** [comment]
Will add Tier 5 Today.

**Comment by Dream-Master** [comment]
Tier 5 and 6 are added. Tier 7 will follow tommorow

**Comment by Dream-Master** [comment]
Done
